lib/interface.js

```diff
@@ -74,7 +103,8 @@ Mocha.interfaces['gagarin'] = module.exports = function (suite) {
         return meteor.stop();
       });
 
-      return meteor;
+      var wrappingMethods = ['execute']
```

Comment:
Good question! This is probably a subset of [these methods](https://github.com/anticoders/gagarin/blob/devel/lib/meteorPromiseChain.js#L40-L58). I think it should be exposed somehow so that we don't need to hardcode the list. I will handle that!